### ISSUE TITLE
`listmaps` command allowed to be called by registered user 0  or 1 but not listed by `adminhelp`

### ISSUE THREAD

**Issue by illwieckz** [Unvanquished/Unvanquished#538]
Hi, registered users level 0 and level 1 can call the command `listmaps` on a server, but `adminhelp` does not list this command.

* calling `adminhelp` command as registered user level 1 lists `listplayers` and `listlayouts` commands and both are allowed, and the hidden `listmaps` command is allowed too.
* calling `adminhelp` command as registered user level 0 lists `listplayers` command only and `listplayers` command is allowed but `listlayouts` command is not allowed, but the hidden `listmaps` command is allowed.

I don’t know if the bug is “_they must no be able to list maps_” or “_adminhelp must list this command_”, but there is a bug for sure. :wink:

**Comment by DolceTriade** [comment]
listmaps is not an admin command.

**Comment by illwieckz** [comment]
Hmm, it's strange to think we can forbid/allow some people to list layouts, but not to list maps…

**Comment by DolceTriade** [comment]
Indeed. Team, what do you think about abandoning different command systems for admin commands and game commands? Is there any benefit to this separation?

**Comment by Kangz** [comment]
+1

**Comment by Viech** [comment]
Sounds like a good idea to me.
